stdlib/public/core/ExistentialCollection.swift.gyb

```diff
@@ -351,19 +351,28 @@ internal class _ForwardIndexBox<
   }
 
   func _distance(to other: _ForwardIndexBoxProtocol) -> AnyForwardIndex.Distance {
-    return numericCast(base.distance(to: unsafeUnbox(other)))
+// FIXME: swift-3-indexing-model - Review what we need to do here
```

Comment:
Yeah I was starting to bang my head against how to do small changes to the Any*Collection code :) ... I will stub as possible to keep things happy until we can get around to working on it.